docs/transforms.md

```diff
@@ -10,11 +10,56 @@ import Model, { attr } from '@ember-data/model';
 export default class PostModel extends Model {
   @attr title;
   @attr('timestamp') createdOn;
+  @attr('timestamp', { defaultValue: null }) modifiedOn;
+  @attr('timestamp', { defaultValue: null }) publishOn;
```

Comment:
These are docs that I feel that we don't need. It should be covered by Ember Data itself.